src/icons/VendorMarker.tsx

```diff
@@ -12,7 +12,7 @@ export interface Props {
   onPress?: () => void
 }
 
-function VendorMarker({ color = colors.onboardingBlue, size = 24, ...props }: Props) {
+function VendorMarker({ color = colors.onboardingBlue, size = 32, ...props }: Props) {
```

Comment:
This will make all vendor markers larger, i thought your intent was to have the selected vendor icon larger?

Reply:
No i felt like the markers themselves were too small, I feel like the icons are in good proportion.